src/config/discord.ts

```diff
@@ -2,5 +2,6 @@ export default {
   token: process.env.DISCORD_TOKEN,
   clientId: process.env.DISCORD_APPLICATION_ID,
   guildId: process.env.DISCORD_SERVER_ID,
+  worker: process.env.DISCORD_WORKER === 'true',
```

Comment:
Seems like a new env variable. Would be good to also add to the README!

Also, is this an optional variable? When is that useful? Maybe dev vs prod?